drivers/display/display_rm67162.c

```diff
@@ -0,0 +1,599 @@
+/*
+ * Copyright 2023, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT raydium_rm67162
+
+#include <zephyr/drivers/display.h>
+#include <zephyr/drivers/mipi_dsi.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/pm/policy.h>
+
+LOG_MODULE_REGISTER(rm67162, CONFIG_DISPLAY_LOG_LEVEL);
+
+/*
+ * These commands are taken from NXP's MCUXpresso SDK.
+ * Additional documentation is added where possible, but the
+ * Manufacture command set pages are not described in the datasheet
+ */
+static const uint8_t rm67162_init_400x392[][2] = {
```

Comment:
You could use an anonymous struct here instead of the second dimension so you could give the two fields a name, then the loop down there would look a it better IMO.

Reply:
Good thought. I've added one here, it is a bit more readable in the loop below.